src/packages/@ncigdc/modern_components/ClinicalAnalysis/boxplot.css

```diff
@@ -15,13 +15,22 @@
   font-size: 1rem;
 }
 
-.boxplot .box line {
-  stroke: #1784AC;
-}
-
 .boxplot .whisker path {
   stroke: #333;
 }
+.demographic-boxplot .box line {
```

Comment:
are these colours stored as constants somewhere?

Reply:
yes, and they'll be passed in as props somehow in another ticket's refactor